packages/bruno-electron/src/index.js

```diff
@@ -24,7 +24,7 @@ const contentSecurityPolicy = [
   "font-src 'self' https:",
   // this has been commented out to make oauth2 work
   // "form-action 'none'",
-  "img-src 'self' blob: data: https:",
+  "img-src 'self' blob: data: http: https:",
   "media-src 'self' blob: data: https:",
   "style-src 'self' 'unsafe-inline' https:"
```

Comment:
Looking at this, I wonder why HTTP would be fine for images but at the same time not for fonts, styles or other media.

I think, if the content security policy is changed, the change should be consistently applied to all kinds.

Reply:
In this case the user can embed an Image from an HTTP-Server in the Docs. But users can't link any fonts etc. And I think we should use the most strict rules if possible.

Reply:
Yes I also thought about it but i didn't want to open security breaches without a need for it. It seems unlikely someone will include a font in the markdown editor, i don't think it's even possible. What do you think?

Reply:
Sorry i didn't see @Its-treason response before so i agree to only allow images because they can be linked in the markdown editor

Reply:
Makes perfect sense. 👍 
Maybe it's worth adding the reasoning to the code as an inline comment. 🙂

Reply:
Ok i can't now but i will later unless you can do it before.
Thanks

Reply:
I suggested a change because I can't directly edit the file.

Reply:
The comment is added now.
Thanks.